js/main.js

```diff
@@ -0,0 +1,178 @@
+const events = [
+  {
+    title: 'Faiz Talk',
+    icon: 'https://raw.githubusercontent.com/RaoAkif/ConferencePage/06f71e60a00948d9ccd7cc8f296e667be6ecf2a9/images/events/remembering_faiz.svg',
+    description:
+      'Listen to the Poets and their beautiful poetry that would enlighten your spirit and revive the era of Faiz Ahmad Faiz.',
+  },
+  {
+    title: 'Ghazal Night',
+    icon: 'https://raw.githubusercontent.com/RaoAkif/ConferencePage/06f71e60a00948d9ccd7cc8f296e667be6ecf2a9/images/events/ghazal_night.svg',
+    description:
+      'Listen to the Poets and their beautiful poetry that would enlighten your spirit and revive the era of Faiz Ahmad Faiz.',
+  },
+  {
+    title: 'Political Debates',
+    icon: 'https://raw.githubusercontent.com/RaoAkif/ConferencePage/06f71e60a00948d9ccd7cc8f296e667be6ecf2a9/images/events/political_debate.svg',
+    description:
+      'Listen to the Poets and their beautiful poetry that would enlighten your spirit and revive the era of Faiz Ahmad Faiz.',
+  },
+  {
+    title: 'Cultural Meetup',
+    icon: 'https://raw.githubusercontent.com/RaoAkif/ConferencePage/06f71e60a00948d9ccd7cc8f296e667be6ecf2a9/images/events/cultural_meetup.svg',
+    description:
+      'Listen to the Poets and their beautiful poetry that would enlighten your spirit and revive the era of Faiz Ahmad Faiz.',
+  },
+  {
+    title: 'Music Night',
+    icon: 'https://raw.githubusercontent.com/RaoAkif/ConferencePage/06f71e60a00948d9ccd7cc8f296e667be6ecf2a9/images/events/music_night.svg',
+    description:
+      'Listen to the Poets and their beautiful poetry that would enlighten your spirit and revive the era of Faiz Ahmad Faiz.',
+  },
+];
+
+const speakers = [
```

Comment:
Awesome ✔ The list of speakers used to generate the HTML is assigned to a variable as an array or object. The same list also contains an speaker object for each that holds their specific data.

Reply:
Thanks